Userland/Applications/Escalator/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+serenity_component(
+    Escalator
+    REQUIRED
+    TARGETS Escalator
+)
+
+compile_gml(Escalator.gml EscalatorGML.h escalator_gml)
+
+set(SOURCES
+    main.cpp
+    EscalatorWindow.cpp
+    EscalatorWindow.h
+    EscalatorGML.h
+)
+
+serenity_app(Escalator ICON app-escalator)
```

Comment:
At least the `32x32` icon does not appear to have been run through `optipng`, which reduces its file size from 1.29KB to 903 bytes.

Reply:
Updated